pkg/volume/cinder/attacher.go

```diff
@@ -44,7 +44,7 @@ var _ volume.Attacher = &cinderDiskAttacher{}
 var _ volume.AttachableVolumePlugin = &cinderPlugin{}
 
 const (
-	checkSleepDuration       = 1 * time.Second
+	checkSleepDuration       = 5 * time.Second
```

Comment:
Should there be an exponential backoff instead of linear? Start with 1 second, then 2, 4, 8, ...

Reply:
Agree that doing this as a backoff is a good idea.

Reply:
Agreed. Most of the time it's fine, but I've seen cases where things get in a weird state where we haven't been able to get things in the right state after multiple retries with increasing backoff times.

Reply:
@jsafrane, @jungleboyj, stmcginnis, Thanks your suggestions.
The latest commit has used an exponential backoff instead of linear.